web/src/lib/components/chat/DropdownGroup.svelte

```diff
@@ -42,6 +63,8 @@
 
   onMount(() => {
     fetchStrategies();
+    sessionAPI.loadSessions();
```

Comment:
This is called without error handling. Compare to the similar code in `SessionManager` (lines 15-21):

```javascript
onMount(async () => {
    try {
      await sessionAPI.loadSessions();
    } catch (error) {
      console.error('Failed to load sessions:', error);
    }
  });
```

So it should be something similar, like:

```javascript
onMount(async () => {
  fetchStrategies();
  try {
    await sessionAPI.loadSessions();
  } catch (error) {
    console.error('Failed to load sessions:', error);
  }
  sessionInput = $currentSession || '';
});
```

In addition, the session handling being in two places (here and in `SessionManager`) can introduce potential race conditions and unnecessary API calls. The session loading should happen once, ideally when the application initializes.